### PR TITLE
Remove k8s_status module, install from galaxy

### PR DESCRIPTION
**Pull request by asmacdo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 - Support for `"ansible.operator-sdk/verbosity"` annotation on Custom Resources watched by Ansible based operators to override verbosity on an individual resource. ([#2102](https://github.com/operator-framework/operator-sdk/pull/2102))
 
 ### Changed
+- Ansible `k8s_status` module has been moved into an [Ansible collection](https://galaxy.ansible.com/operator_sdk/util).
+- Upgrade minimal Ansible version in the init projects from `2.6` to `2.9`. ([TODO](TODO))
```
Comment (camilamacedo86) [inline CHANGELOG.md]: nit ^
```suggestion
- Upgrade minimal Ansible version in the init projects from `2.6` to `2.9`. ([#2310](https://github.com/operator-framework/operator-sdk/pull/2310))
```

File: doc/ansible/dev/developer_guide.md [hunk 2]
```diff
@@ -378,10 +382,13 @@ from your application, then simply update the watches file with `manageStatus`:
...
-To update the `status` subresource with key `foo` and value `bar`, `k8s_status`
-can be used as shown:
+The simplest way to invoke the `k8s_status` module is to
+use its fully qualified collection name (fqcn). To update the
+`status` subresource with key `foo` and value `bar`, `k8s_status` can be
+used as shown:
+
 ```yaml
-- k8s_status:
+- operator_sdk.util.k8s_status:
```
Comment (camilamacedo86) [inline doc/ansible/dev/developer_guide.md]: It shows a break change. Then, in POV this info needs to be signalized in the CHANGELOG.

File: doc/ansible/dev/developer_guide.md [hunk 3]
```diff
@@ -423,23 +448,12 @@ the Ansible Operator, see the [proposal for user-driven status
...
-```
-
-Add `k8s_status.py` to `/usr/share/ansible/library/`:
-```bash
-$ wget https://raw.githubusercontent.com/fabianvf/ansible-k8s-status-module/master/k8s_status.py -O /usr/share/ansible/library/k8s_status.py
-```
+interested in testing the operator with `operator-sdk up local`, then
+you will need to install the collection locally.
 
+``` bash
```
Comment (estroz) [inline doc/ansible/dev/developer_guide.md]:
```suggestion
```sh
```

File: doc/ansible/dev/developer_guide.md [hunk 4]
```diff
@@ -423,23 +448,12 @@ the Ansible Operator, see the [proposal for user-driven status
...
+ ansible-galaxy collection install operator_sdk.util
```
Comment (estroz) [inline doc/ansible/dev/developer_guide.md]:
```suggestion
$ ansible-galaxy collection install operator_sdk.util
```

File: internal/scaffold/ansible/roles_meta_main.go [hunk 5]
```diff
@@ -97,4 +97,6 @@ const rolesMetaMainAnsibleTmpl = `galaxy_info:
 
 dependencies: []
   # List your role dependencies here, one per line. Be sure to remove the '[]' above,
-  # if you add dependencies to this list.`
+  # if you add dependencies to this list.
+collections:
+- operator_sdk.util`
```
Comment (joelanford) [inline internal/scaffold/ansible/roles_meta_main.go]: Is there a best practice between declaring the collection like this and then using the shortened name in tasks vs using the fully qualified name in tasks?

Comment (asmacdo) [reply internal/scaffold/ansible/roles_meta_main.go]: They don't seem to have a preference yet.
https://docs.ansible.com/ansible/devel/user_guide/collections_using.html#using-collections-in-a-playbook

**Pull request merged**